generator/template/ready-to-use-section-types/components/ResturantMenu/ShoppingCart.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <div class="fixed inset-0 bg-black bg-opacity-50 z-50" @click="$emit('close')">
+    <div
+      class="absolute right-0 top-0 h-full bg-white w-full max-w-md overflow-y-auto shadow-xl transform transition-transform duration-300"
+      @click.stop>
+      <div class="p-6">
+        <div class="flex justify-between items-center mb-6">
+          <h2 class="text-xl font-bold">{{ $t('RestaurantMenu.cart') || 'Your Cart' }}</h2>
+          <div @click="$emit('close')" class="p-2 rounded-full hover:bg-gray-200">
+            <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24"
+              stroke="currentColor" width="24" height="24">
+              <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12" />
+            </svg>
+          </div>
+        </div>
+
+        <!-- Empty cart message -->
+        <div v-if="cart.length === 0" class="text-center py-8">
+          <svg xmlns="http://www.w3.org/2000/svg" class="h-16 w-16 mx-auto text-gray-400 mb-4" fill="none"
+            viewBox="0 0 24 24" stroke="currentColor" width="64" height="64">
+            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+              d="M3 3h2l.4 2M7 13h10l4-8H5.4M7 13L5.4 5M7 13l-2.293 2.293c-.63.63-.184 1.707.707 1.707H17m0 0a2 2 0 100 4 2 2 0 000-4zm-8 2a2 2 0 11-4 0 2 2 0 014 0z" />
+          </svg>
+          <p class="text-gray-600">{{ $t('RestaurantMenu.emptyCart') || 'Your cart is empty' }}</p>
```

Comment:
Same for the conditions not valid in this component, better to have default translations for these

Reply:
already supported but removed them